L06-annotations/src/main/java/ru/otus/framework/TestClass.java

```diff
@@ -0,0 +1,36 @@
+package ru.otus.framework;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import ru.otus.framework.annotations.After;
+import ru.otus.framework.annotations.Before;
+import ru.otus.framework.annotations.Test;
+
+public class TestClass {
+    private static final Logger logger
+            = LoggerFactory.getLogger(TestClass.class);
+@Before
+    void setUp() {
+        logger.info("setting up before each test");
+    }
+@After
+    void shutDown() {
+        logger.info("shut down after each test");
+    }
+
```

Comment:
Странное форматирование кода - аннотация слева без отступов.
```suggestion
    @After
    void shutDown() {
        logger.info("shut down after each test");
    }

```